chirp/drivers/ga510.py

```diff
@@ -795,19 +1022,6 @@ def get_features(self):
                           (400000000, 520000000)]
         return rf
 
-    def validate_memory(self, mem):
-        msgs = super().validate_memory(mem)
-
-        _msg_duplex = 'Duplex must be "off" for this frequency'
-        _msg_offset = 'Only simplex or +5MHz offset allowed on GMRS'
-
-        if not ((mem.freq >= self.vhftx[0] and mem.freq < self.vhftx[1]) or
-                (mem.freq >= self.uhftx[0] and mem.freq < self.uhftx[1])):
-            if mem.duplex != "off":
-                msgs.append(chirp_common.ValidationWarning(_msg_duplex))
-
-        return msgs
-
```

Comment:
As noted above, please don't move and combine these. @KC9HI has *just* gotten these validations correct so that they happen in the relevant classes and with the relevant rules. If your radios have the same rules, then perhaps they can share an implementation.